### DISCUSSION TITLE
Is there a way to get anything from one device to another?

### DISCUSSION THREAD

**Discussion by MustardMonday** [keiwando/toonsquid#701] [Q&A]
Hi, I have an older device which I prefer to use toonsquid on for one reason or another, but I’m scared that since it’s old it’s breaking down a little, I had some unfinished things I was working on on that device and I’m afraid that it’s gone flat and tried charging but won’t come back on. I’ll try different characters but that’s not the point, is there a way to get my unfinished things back? Like an account feature maybe? If not please add one! I have a habit of not saving things and it seems to caught up to me.

**Comment by Cat-Butler** [comment]
If your iPad has been making automatic iCloud backups, you can [restore from that backup, to a new or newly erased iPad](https://support.apple.com/en-au/guide/ipad/ipad9a74c804/17.0/ipados/17.0).
Might take several days in some cases, depending on backup size, network speeds, etc.

Otherwise, manual export to .tsproj files is the best way to go, to some sort of cloud service (iCloud, Dropbox, Google Drive, OneDrive, etc).

If you can get access to the iPad storage (e.g. will the iPad run while it's plugged in to power?), you might be able to access your projects either via the ToonSquid interface or the ToonSquid Project and Backup folders .
